# Размер данных которые необходимо отправить
int32 num

# Строка, которую хотим отправить
string data